lib/src/data/datasource/local/DAOs/card_database.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import '../../../../core/util/constants_strings.dart';
+// ignore_for_file: avoid_dynamic_calls
```

Comment:
This is necessary?

Reply:
I couldn't work around it. Asked Leo and he told me it was okay to do so. We can check it out